module/config/actor.mjs

```diff
@@ -137,6 +137,22 @@ export const TRAVEL_PACES = freezeEnum({
   }
 });
 
+/**
+ * Categories a language can (optionally) belong to
+ * @type {Record<string, string>}
```

Comment:
I would prefer we go with:
```suggestion
 * @type {Record<string, {label: string}>}
```
This gives us more room to grow in case we ever need other category metadata